pkg/util/util.go

```diff
@@ -293,3 +293,24 @@ func GenerateKubeVirtGroupVersionKind(obj runtime.Object) (runtime.Object, error
 
 	return objCopy, nil
 }
+
+/*
+TranslateBuildArch translates the build_arch to arch
+
+	case1:
+	  build_arch is crossbuild-s390x, which will be translated to s390x arch
+	case2:
+	  build_arch is s390x, which will be translated to s390x arch
+*/
+func TranslateBuildArch() string {
```

Comment:
Do we expect to have this env during execution of the test?

Reply:
Yes

Reply:
We run the binary directly, what will set the env?

Reply:
This env will be set in the prow job that runs the e2e test for s390x.  We will set this env to `BUILD_ARCH="crossbuild-s390x"`